cli/lib/validation.ts

```diff
@@ -17,3 +19,45 @@ function quadToString(quad: Quad): string {
     graph: quad.graph as NamedNode,
   })
 }
+
+// consider replacing should with may
```

Comment:
for example, the message for `xsd:dateTime` is "**should** be formatted as YYYY-MM-DDThh:mm:ss", but the spec allows for many more variations; describing all of them in a simple yet precise way is challenging, that's why **may** instead of **should** may be more appropriate

Reply:
I think **should** is appropriate :)